infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java

```diff
@@ -117,14 +118,13 @@
     private static final CharSequence NUMBER_OF_NODES_PATTERN = "<NUMBER_OF_NODES>";
 
     // Command lines definition
-    private static final String POWERSHELL_DOWNLOAD_CMD = "powershell -command \"& { (New-Object Net.WebClient).DownloadFile('http://" +
-                                                          RM_HOSTNAME_PATTERN + ":8080/rest/node.jar" +
-                                                          "', 'node.jar') }\"";
+    private static final String POWERSHELL_DOWNLOAD_CMD = "$wc = New-Object System.Net.WebClient; $wc.DownloadFile('http://" +
```

Comment:
We will have an issue if the scheduler is deployed in https and/or with a different port